inference-engine/src/vpu/myriad_plugin/myriad_config.cpp

```diff
@@ -95,6 +97,15 @@ IE_SUPPRESS_DEPRECATED_END
         { VPU_MYRIAD_CONFIG_VALUE(POWER_STAGE_NCES),   PowerConfig::STAGE_NCES },
     };
 
+    static const std::unordered_map<std::string, char> memoryTypes = {
+        { VPU_MYRIAD_CONFIG_VALUE(NOT_PROG),        0 },
+        { VPU_MYRIAD_CONFIG_VALUE(MA2485A1),        1 },
+        { VPU_MYRIAD_CONFIG_VALUE(MA2485A1B),       2 },
+        { VPU_MYRIAD_CONFIG_VALUE(MA2485B0C0),      3 },
+        { VPU_MYRIAD_CONFIG_VALUE(MA2085C0),        4 },
+        { VPU_MYRIAD_CONFIG_VALUE(MA2085C0_PRIME),  5 }
```

Comment:
Should there be used `enum class` containing members with descriptive names instead of `char`?

Reply:
I wanted avoid type conversion between plugin and mvnc, but maybe you are right..